scripts/openmc-make-stopping-powers

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+from urllib.parse import urlencode
+from urllib.request import urlopen
+from lxml import html
+
+import numpy as np
+import h5py
+from openmc.data import ATOMIC_SYMBOL
+
+
+base_url = 'https://physics.nist.gov/cgi-bin/Star/e_table-t.pl'
+energies = np.logspace(-3, 3, 200)
+data = {'matno': '', 'Energies': '\n'.join(str(x) for x in energies)}
+columns = {1: 's_collision', 2: 's_radiative'}
+
+# ==============================================================================
+# SCRAPE DATA FROM ESTAR SITE AND GENERATE STOPPING POWER HDF5 FILE
+
+print('Generating stopping_powers.h5...')
+
+with h5py.File('stopping_powers.h5', 'w') as f:
+
+    # Write energies
+    f.create_dataset('energy', data=energies)
+
+    for Z in range(1, 99):
```

Comment:
In other files loops over atomic numbers range from elements 1-100, is it intentional that in this case the range is from 1-98?